### ISSUE TITLE
ropsten  Network Error

### ISSUE THREAD

**Issue by ezailWang** [sebs/etherscan-api#31]
`etherscan.init(ETHERSCAN_API_KEY, 'ropsten', 10000).contract.getabi(address)`
Not only to call this method, other methods will also report the error, other test network or the main network has no problem
Does ropsten test network not work?

**Comment by sebs** [comment]
can you give me a complete call please, including a valid contract address. I will investigate.

**Comment by ezailWang** [comment]
```
var api = etherscan.init(ETHERSCAN_API_KEY, 'ropsten', 10000)
api.contract.getabi('0x4470A46A561C78971252c26c5FF036bB83B2DD42')
```

**Comment by ezailWang** [comment]
```
static async getERC20Transations(contractAddress, decimals) {
        const { walletAddress } = store.getState().Core
        var data = await api.account.tokentx(walletAddress, contractAddress)
        if (data.message !== 'OK') {
            return []
        }
        return data.result.map(t => ({
            from: t.from,
            timeStamp: t.timeStamp,
            hash: t.hash,
            value: (parseInt(t.value, 16) / Math.pow(10, decimals)).toFixed(2),
        }))
    }
```
Obtaining transaction records will also report this error

**Comment by sebs** [comment]
This just gets me an error from the backend, I have to point you to etherscan directly. I am just getting a 'NOT OK' with no further response from the server.

**Comment by sebs** [comment]
Ah I am not parsing the full error, which now is where the result was. I need to come up with an error handler

**Comment by sebs** [comment]
I have released 9.0.0. 

This presents the right error message
